.github/workflows/ci.yml

```diff
@@ -210,12 +210,10 @@ jobs:
           export LD_LIBRARY_PATH=${HOME}/lib/$(uname -m)-linux-gnu:${HOME}/lib:${HOME}/lib64:${LD_LIBRARY_PATH}
           export PKG_CONFIG_PATH=${HOME}/lib/pkgconfig:${HOME}/lib/$(uname -m)-linux-gnu/pkgconfig:${PKG_CONFIG_PATH}
           ninja -C build install
-      - name: Disable user authentication for debugging (MacOS)
+      - name: Disable user authentication for debugging (macOS)
         if: matrix.os == 'macos-11' && matrix.enabled
         run: |
-          security authorizationdb read system.privilege.taskport > taskport.plist
```

Comment:
It is, but it can be simplified to not having to edit the plist manually. The `sudo security authorizationdb write system.privilege.taskport allow` opens it up further but it doesn't matter since we have root anyway.